firmware/src/model/stockDataModel.cpp

```diff
@@ -15,6 +15,34 @@ StockDataModel &StockDataModel::setSymbol(String symbol) {
 String StockDataModel::getSymbol() {
     return m_symbol;
 }
+
+
+
+
+StockDataModel &StockDataModel::setCompany(String company) {
+    if (m_company != company) {
+        m_company = String(company);
```

Comment:
Why the explicit cast to `String`? It is already a `String`

Reply:
Was just re-using code, fixed.

Reply:
Also fixed for currency